TwitchLib.PubSub/Models/Responses/Message.cs

```diff
@@ -50,6 +50,12 @@ public Message(string jsonStr)
                 case "following":
                     MessageData = new Following(encodedJsonMessage);
                     break;
+                case "community-points-channel-v1":
+                    MessageData = new CommunityPointsChannel(encodedJsonMessage);
+                    break;
+                case "leaderboard-events-v1":
+                    MessageData = new LeaderboardEvents(encodedJsonMessage);
+                    break;
```

Comment:
These are presumably undocumented (I didn't see them in the docs). We might want to note that somewhere going forward.

Reply:
Where can i add some notes to this feature? Its a weekly leaderboard for subgifts and bits.